pmlc/runtime/llvm/BUILD

```diff
@@ -3,20 +3,15 @@
 load("//bzl:plaidml.bzl", "plaidml_cc_library")
 
 plaidml_cc_library(
-    name = "rt",
+    name = "llvm",
     srcs = [
-        "boundscheck.cc",
+        "builtin.cc",
         "runtime.cc",
-        "xsmm.cc",
-        "prng.cc",
     ],
-    visibility = ["//visibility:public"],
     deps = [
         "//pmlc/compiler",
-        "@half",
-        "@llvm-project//llvm:Support",
-        "@llvm-project//mlir:mlir_runner_utils",
-        "@xsmm",
```

Comment:
Why isn't `@xsmm` specific to the `x86` runtime? It's not really an LLVM runtime, it's more like an `x86`-specific runtime.